tests/test_flow/test_persistence_aip.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+import bionic as bn
+
+
+# This is detected by pytest and applied to all the tests in this module.
+pytestmark = [pytest.mark.needs_aip, pytest.mark.needs_gcs]
+
+
+def test_aip_jobs(aip_builder):
```

Comment:
Is there a way we can test that this actually uses AIP?

Reply:
Once we have a different task implementation of test, we can run this test using both the implementations. We can run the test implementation by default, and run the AIP implementation on our local machines using some kind of flag. Like I mentioned in an earlier [reply](https://github.com/square/bionic/pull/253#discussion_r487122003), I'm waiting for the fix in the base PR, once that's done, I'll change the test to do as I described above.

I guess I could've made it clear in the description or waited till the base PR had the fixes in. My bad.